public/docs/ts/latest/guide/glossary.jade

```diff
@@ -0,0 +1,445 @@
+:markdown
+  # Angular 2 Glossary
+#sg-tables.showcase.shadow-1
+  header.showcase-header
+    p.
+      <i>The difference between the right word
+      and the almost right word is the difference between
+      lightening and a lightening bug.</i> - Mark Twain
+
+:markdown
+  Angular 2 has a vocabulary of its own.
+  Most Angular 2 terms are everyday English words 
+  with a specific meaning within the Angular system.
+  
+  We have gathered here the most prominent terms
+  and a few less familiar ones that have unusual or
+  unexpected definitions.
+
+.l-main-section
+:markdown
+  ## Annotation
+.l-sub-section
+  :markdown
+    In practice a synonym for [Decoration](#decoration).
+    
+ :markdown
+  ## Attribute Directive
+.l-sub-section
+  :markdown
+    A category of [Directive](#directive) that can listen to and modify the behavior of
+    other HTML elements, attributes, properties, and components. They are usually represented
+    as HTML attributes, hence the name.
+    
+    The `ng-class` directive for adding and removing CSS class names is a good example of
+    an Attribute Directive.
+
+.l-main-section
+:markdown
+  ## Binding
+.l-sub-section
+  :markdown
+    Almost always refers to [Data Binding](#data-binding) and the act of
+    binding an HTML object property to a data object property.
+     
+    May refer to a [Dependency Injection](#dependency-injection) binding
+    between a "token" or "key" and a dependency [provider](#provider).
+    This more rare usage should be clear in context.
+    
+:markdown
+  ## Bootstrap
+.l-sub-section
+  :markdown
+    We launch an Angular application by "bootstrapping" it with the `bootstrap` method.
+    The `bootstrap` method identifies the application's' top level "root" [Component](#component)
```

Comment:
It is true that one can bootstrap multiple apps. That isn't a typical scenario but it is possible and even useful.

I'd like to see a chapter on bootstrap that I can refer to rather than complicate things here. Open to alternative language of course!